Author: Michael Biebl <biebl@debian.org>
Date:   Mon Nov 10 13:33:48 2008 +0100

    allow uid/gid mount option for ntfs

    Allow uid/gid mount option for ntfs, as reported in
    http://bugs.debian.org/427109.

diff -Nur hal-0.5.11.orig/tools/hal-storage-mount.c hal-0.5.11/tools/hal-storage-mount.c
--- hal-0.5.11.orig/tools/hal-storage-mount.c	2008-05-07 18:24:23.000000000 -0500
+++ hal-0.5.11/tools/hal-storage-mount.c	2009-07-16 16:13:12.544551085 -0500
@@ -720,6 +720,8 @@
 		 * (since these doesn't contain uid/gid bits) 
 		 */
 		if (strcmp (libhal_volume_get_fstype (volume), "vfat") != 0 &&
+		    strcmp (libhal_volume_get_fstype (volume), "ntfs") != 0 &&
+		    strcmp (libhal_volume_get_fstype (volume), "ntfs-3g") != 0 &&
 		    strcmp (libhal_volume_get_fstype (volume), "iso9660") != 0 &&
 		    strcmp (libhal_volume_get_fstype (volume), "hfs") != 0 &&
 		    strcmp (libhal_volume_get_fstype (volume), "udf") != 0) {
